### ISSUE TITLE
Unexpected generic wrapping with `wrap`

### ISSUE THREAD

**Issue by keith** [nicklockwood/SwiftFormat#647]
With this code:

```swift
final class VehicleDamageLocationCardViewController: GeneralCardViewController<
    VehicleDamageLocationInputRecordBinding>
{
    override func syncUI() {
        super.syncUI()

        self.cardView.configure(questionPrompt: self.binding.question,
                                selectedZone: self.binding.answer,
                                accessibilityIdentifier: self.binding.accessibilityIdentifier)
    }
}
```

If you format it with `swiftformat --rules wrap --maxwidth 110 foo.swift --swiftversion 5.2 --cache clear` you end up with:

```swift
final class VehicleDamageLocationCardViewController: GeneralCardViewController<VehicleDamageLocationInputRecordBinding>
{
    override func syncUI() {
        super.syncUI()

        self.cardView.configure(questionPrompt: self.binding.question,
                                selectedZone: self.binding.answer,
                                accessibilityIdentifier: self.binding.accessibilityIdentifier)
    }
}

```

Where the first line now exceeds the 110 width. If I remove the `syncUI` function I end up with more what I would expect:

```
final class VehicleDamageLocationCardViewController: GeneralCardViewController<
    VehicleDamageLocationInputRecordBinding
>
{
}
```

0.44.13

**Comment by nicklockwood** [comment]
@keith fixed in 0.44.15. Thanks for reporting!

**Comment by keith** [comment]
Thanks for fixing all these!!
